rbd/admin/admin.go

```diff
@@ -28,11 +28,15 @@ type LevelSpec struct {
 }
 
 // NewLevelSpec is used to construct a LevelSpec given a pool and
-// optional namespace and image names.
-func NewLevelSpec(pool, namespace, image string) LevelSpec {
+// optional namespace, image and group names.
+func NewLevelSpec(pool, namespace, image, group string) LevelSpec {
```

Comment:
@Nikhil-Ladha Lets add a new function for the group instead of adding it to NewLevelSpec because group and image doesnt go along both are different and both cannot be set at the same time.